net2/SysManager.js

```diff
@@ -73,6 +73,7 @@ const f = require('../net2/Firewalla.js');
 const i18n = require('../util/i18n.js');
 
 const dns = require('dns');
+const { reduce } = require('bluebird');
```

Comment:
no need to import special reduce function, nodejs has native reduce

Reply:
Removed as suggested

Reply:
Removed as suggested